examples/pipeline/kg_builder_with_pdf_loader.py

```diff
@@ -0,0 +1,159 @@
+#  Copyright (c) "Neo4j"
+#  Neo4j Sweden AB [https://neo4j.com]
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      https://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from __future__ import annotations
```

Comment:
Maybe we can also just update the `kg_builder.py` file, so that we only have one end-to-end example?

Reply:
Hmm I changed it such that there is one example from PDF (using the Harry Potter file), and one example from text (using the Albert Einstein text). I also deleted `kg_builder_with_schema.py`. Does that work?

Reply:
Sounds good!